packages/components/organisms/f-offers/src/store/types/mutations.js

```diff
@@ -0,0 +1,3 @@
+const EMIT_SEARCH_START = 'searchbox-focus';
+
+export default EMIT_SEARCH_START;
```

Comment:
It seems strange to me to be exporting as default a single value from a file that seems designed to export multiple as per `actions.js` above, even if for the moment it only contains one - is this to avoid a linting warning?

Reply:
yes this is to avoid a linting warning, I could add a lint disable if that would be ok for this situation.

Reply:
We had a similar issue on menu where we simply added an eslint disable comment until more exports were added that allowed us to keep these types of files consistent

```js
/* eslint-disable import/prefer-default-export */

export const EMIT_SEARCH_START = 'searchbox-focus';
```

Reply:
I have added a types file and namespaced as I liked that idea @benwhite-deltas